README.md

```diff
@@ -39,4 +39,22 @@ The Fileglancer Central service is a backend service optionally used by Fileglan
 ![Fileglancer Architecture drawio](https://github.com/user-attachments/assets/216353d2-082d-4292-a2eb-b72004087110)
 
 
+## Building the Docker container
+
+Run the Docker build on a Linux x86 system, replacing `<version>` with your version number:
+
+```bash
+cd docker/
+export VERSION=<version>
+docker build . --build-arg GIT_TAG=$VERSION -t ghcr.io/janeliascicomp/fileglancer-central:$VERSION
+```
+
+## Pushing the Docker container
+
+```bash
+docker push ghcr.io/janeliascicomp/fileglancer-central:$VERSION
+docker tag ghcr.io/janeliascicomp/fileglancer-central:$VERSION ghcr.io/janeliascicomp/fileglancer-central:latest
+docker push ghcr.io/janeliascicomp/fileglancer-central:latest
+```
+
```

Comment:
We could do all of this with github actions. Do you want to add that as well as the instructions here?

Reply:
Not sure I follow. What instructions?

Reply:
Do you want to create the gihub action that will build the image and place it in ghcr.io when you push a tagged version? The "instructions" I was talking about are the ones that you have here on how to build the image, which would be useful, but somewhat redundant if the github action is doing it for you.

Reply:
Here is an example from constructDB:
[https://github.com/JaneliaSciComp/ConstructDB/blob/master/.github/workflows/publish-docker-image.yml](https://github.com/JaneliaSciComp/ConstructDB/blob/master/.github/workflows/publish-docker-image.yml)

Reply:
That looks great, I'll try to get to it.